polkadot-parachains/src/command.rs

```diff
@@ -332,15 +316,7 @@ pub fn run() -> Result<()> {
 			match cmd {
 				BenchmarkCmd::Pallet(cmd) =>
 					if cfg!(feature = "runtime-benchmarks") {
-						runner.sync_run(|config| {
-							if config.chain_spec.is_launch() {
-								cmd.run::<Block, LaunchParachainRuntimeExecutor>(config)
-							} else if config.chain_spec.is_encointer() {
-								cmd.run::<Block, EncointerParachainRuntimeExecutor>(config)
-							} else {
-								Err("Chain doesn't support benchmarking".into())
-							}
-						})
```

Comment:
hmm, this remove doesn't look sane neither

Reply:
no, actually I think this is no longer needed as native runtime execution has been removed, right?

Reply:
if it should be needed, we must implement trait HostFunctions.....

Reply:
I was surprised too, but then I thought if the upstream passes `()`, it must just be a simplification from upstream, and as it is only the benchmarks, it does make sense to me intuitively that you actually don't need an executor.